data/nlu/nlu.yml

```diff
@@ -231,6 +231,13 @@ nlu:
     - to my [brother](PERSON)
     - to my [sister](PERSON)
     - my [dad](PERSON)
+    - William Baker
```

Comment:
oh hm, i'm wondering if the other data should not be annotated with PERSON -- maybe rasa x is saving that from the nlu inbox accidentatlly?

Reply:
No I think that's Arjaan's work to allow Diet to catch things that Spacy doesn't. Names can be left alone but spacy doesn't extract friend, mother, etc.

Reply:
Ah okay that makes sense!